src/vs/editor/browser/controller/editContext/native/nativeEditContext.ts

```diff
@@ -406,7 +407,13 @@ export class NativeEditContext extends AbstractEditContext {
 		// system caret. This is reflected in Chrome as a `selectionchange` event and needs to be reflected within the editor.
 
 		return addDisposableListener(this.domNode.domNode.ownerDocument, 'selectionchange', () => {
-			if (!this.isFocused()) {
+			const options = this._context.configuration.options;
+			const accessibilitySupportEnabled = options.get(EditorOption.accessibilitySupport);
```

Comment:
Should you be using `IAccessibilityService` for this?

Reply:
Ah yes I see there is a method which checks if accessibility mode is enabled. I will use that instead.